ngraph/src/ngraph/pass/pass.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace std;
 using namespace ngraph;
 
+NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
Why do we need it here?

Reply:
It is needed to fix windows build which is failed on function definition.